DEVELOPMENT.md

```diff
@@ -202,3 +202,41 @@ Additional information which is really important to how the application runs can
 be found in the manifest:
 
 > `$ bundle exec 3scale_backend manifest`
+
+## Running with RubyMine
+
+Using RubyMine scripts can help you run both Worker and Listener.
```

Comment:
I don't like mentioning RubyMine in particular because it's something we use but the docs are for the public and I don't think much people use RubyMine since it's paid and not free software.

I suggest this rewording:

```suggestion

## Running from your IDE

Some IDEs like VSCode or RubyMine allow to add custom launchers for scripts. This is convenient to launch both Worker and Listener.
```

Reply:
Sounds good, applied!